include/topology.h

```diff
@@ -1102,12 +1102,12 @@ typedef struct snd_tplg_obj_template {
 } snd_tplg_obj_template_t;
 
 /**
- * \brief Register topology template object.
+ * \brief Register topology template element.
  * \param tplg Topology instance.
- * \param t Template object.
+ * \param t Template element.
  * \return Zero on success, otherwise a negative error code
  */
-int snd_tplg_add_object(snd_tplg_t *tplg, snd_tplg_obj_template_t *t);
+int snd_tplg_add_element(snd_tplg_t *tplg, snd_tplg_obj_template_t *t);
```

Comment:
I think that it's a bit late to rename the exported function. You may use 'cinstance' (class instance) or another name for the objects defined in the configuration.